### ISSUE TITLE
MS Teams screen share not working between two Ubuntu Users

### ISSUE THREAD

**Issue by fernandobalieiro** [ferdium/ferdium-app#2003]
### Avoid duplicates

- [X] I agree to follow the [Code of Conduct](https://github.com/ferdium/ferdium-app/blob/HEAD/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issues tracker](https://github.com/ferdium/ferdium-app/issues?q=is%3Aopen+is%3Aissue+label%3A%22bug+%3Abeetle%3A%22) for a bug report similar to mine, in vain

### Ferdium Version

7.0.0

### What Operating System are you using?

Ubuntu

### Operating System Version

Linux 6.5.0-1023-oem #24-Ubuntu SMP PREEMPT_DYNAMIC Tue May  7 14:26:31 UTC 2024 x86_64 x86_64 x86_64 GNU/Linux

### What arch are you using?

x64

### Last Known Working Ferdium version

6.x.x?

### Expected Behavior

MS Teams screen share should work between two Ubuntu users.

### Actual Behavior

I have a colleague that also uses Ferdium on Ubuntu. When one of us is sharing their screen on MS Teams, the other one cannot see it. It does not matter if we have more people in the call or not, the issue remains the same.

If the one that is sharing their screen, share it on MS Teams **in a browser** instead, it works fine, even when the other (the one that is receiving the screen share) is using Ferdium.

### Steps to reproduce

1. Two Ubuntu Users join the same call on MS Teams;
2. User 1 share their screen;
3. User 2 cannot see the shared screen.

### Screenshots

Screen shared by another Ubuntu User:
![image](https://github.com/user-attachments/assets/43690f1d-d3f9-4a54-802d-11c80fb2cdd2)

### Additional information

We are not totally sure when this stopped working, although it seems that working normally a few weeks ago, maybe on version 6.x.x?

**Comment by bukajsytlos** [comment]
imho it does not matter who is on the other side. just sharing on ubuntu/snap/wayland is not working. it won't offer desktop to share unlike in browser.

version: 6.7.7

**Comment by ollosh** [comment]
I'm experiencing the same problem. Everyone who's on Linux for some reason cannot see my screen, while people on Mac's and Windows' can 🤯 I did also notice that it happens in a call where multiple people are while in a call where it's only 2 people it is usually ok

**Comment by juanfraherrero** [comment]
Yep, in a 1:1 the sharing is visible, but in a call with multiple users the share screen keeps gray.

Version: 7.1.0
Server: No Server
Electron: 35.4.0
Chrome: 134.0.6998.205
Node.js: 22.15.0
Platform: Linux 6.15
Arch: x64
